application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -128,6 +128,9 @@
 ## Check if the document needs migration
 
 #set ($type = $pollObj.getValue('type'))
+#if ("$!type" == '')
+  #set ($discard = $pollObj.set('type', 'multi'))
```

Comment:
Shouldn't you also set `type` to have the `multi` value? Isn't it empty right now?